gocse/helpers/aws.go

```diff
@@ -0,0 +1,68 @@
+package helpers
+
+import (
+	"log"
+	"os"
+
+	"github.com/fatih/structs"
+	"go.mongodb.org/mongo-driver/bson"
+)
+
+// KMS Providers, the credentials used to access the KMS provider
```

Comment:
I think the text "KMS Providers" is ambiguous here (and every other occurence); maybe you meant something like KMS Provider Credentials?.